dev/docs/migrating-v1beta1-to-v1.md

```diff
@@ -35,6 +35,13 @@ In Tekton `v1`, the following fields have been changed:
 | `pipelineRun.spec.resources` | removed from `PipelineRun` |
 | `pipelineRun.spec.serviceAccountName` | [`pipelineRun.spec.taskRunTemplate.serviceAccountName`](#adding-taskruntemplate-to-pipelinerun.spec) |
 | `pipelineRun.spec.podTemplate` | [`pipelineRun.spec.taskRunTemplate.podTemplate`](#adding-taskruntemplate-to-pipelinerun.spec) |
+| `task.spec.steps[].resources` | `task.spec.steps[].computeResources` |
+| `task.spec.stepTemplate.resources` | `task.spec.stepTemplate.computeResources` |
+| `task.spec.sidecars[].resources` | `task.spec.sidecars[].computeResources` |
+| `taskRun.spec.stepOverrides[].resources` | `taskRun.spec.stepOverrides[].computeResources` |
+| `taskRun.spec.sidecarOverrides[].resources` | `taskRun.spec.sidecarOverrides[].computeResources` |
```

Comment:
why are `stepOverrides` and `sidecarOverrides` not renamed here but they are renamed to `stepSpecs` and `sidecarSpecs` below? looks like typos based on the implementation

Reply:
Thanks @jerop. In this line I tried to suggest that we're changing the `resources` field to `computeResources`. I'm afraid it might be confusing if it's one step to `taskRun.Spec.sidecarSpecs[].computeResources`, so I attempted to show that we're changing sidecarOverrides field to sidecarSpecs below. Does that make sense?

Reply:
what if you show `overrides` to `specs` rename first, ahead of the `resources` to `computeResources`? i.e. move L43-L44 ahead of L41-L42

that way, we're showing what the full replacement of the field is

Reply:
Thank you, Jerop! That makes sense, updated.